services/ffc-demo/base/kustomize.yaml

```diff
@@ -14,7 +14,7 @@ spec:
   path: ./services/ffc-demo/${ENVIRONMENT}
   postBuild:
     substitute:
-      NAMESPACE: "adp-demo"
+      NAMESPACE: "ffc-demo"
```

Comment:
adp denmo?

Reply:
@DaRoszko : Discussed with Ken.
At the moment application helm chart image is still using 'ffc-demo' namespace and we can not use 'adp-demo' from flux-service until necessary changes made in application manifest/helmcharts. So for now we just decided to stay with ffc-demo and will make necessary changes in upcoming Pr's